test/archethic_test.exs

```diff
@@ -1,33 +1,158 @@
 defmodule ArchethicTest do
   use ArchethicCase
 
-  alias Archethic.Crypto
-
-  alias Archethic.PubSub
-
-  alias Archethic.P2P
-  alias Archethic.P2P.Message.Balance
-  alias Archethic.P2P.Message.GetBalance
-  alias Archethic.P2P.Message.GetLastTransactionAddress
-  alias Archethic.P2P.Message.GetTransaction
-  alias Archethic.P2P.Message.GetTransactionChainLength
-  alias Archethic.P2P.Message.GetTransactionInputs
-
-  alias Archethic.P2P.Message.LastTransactionAddress
-  alias Archethic.P2P.Message.NotFound
-  alias Archethic.P2P.Message.Ok
-  alias Archethic.P2P.Message.StartMining
-  alias Archethic.P2P.Message.TransactionChainLength
-  alias Archethic.P2P.Message.TransactionInputList
-  alias Archethic.P2P.Node
-
-  alias Archethic.TransactionChain.Transaction
-  alias Archethic.TransactionChain.TransactionData
-  alias Archethic.TransactionChain.TransactionInput
-  alias Archethic.TransactionChain.VersionedTransactionInput
+  alias Archethic.{Crypto, PubSub, P2P, P2P.Message, P2P.Node, TransactionChain, SelfRepair}
+  alias Archethic.{BeaconChain.SummaryTimer}
+
+  alias Message.{Balance, GetBalance, GetLastTransactionAddress, GetTransaction, Ok}
+  alias Message.{GetTransactionChainLength, GetTransactionInputs, LastTransactionAddress}
+  alias Message.{NotFound, StartMining, TransactionChainLength, TransactionInputList}
+  alias Message.{NewTransaction}
+
+  alias TransactionChain.{Transaction, TransactionData}
+  alias TransactionChain.{TransactionInput, VersionedTransactionInput}
 
   import Mox
 
+  setup do
+    nss_genesis_address = "nss_genesis_address"
```

Comment:
is it possible to add this setup as a named setup in ArchethicCase and use `setup [:nss_mock_setup]` to reference it where you need it?